aws_queries/scheduled_rule_default.py

```diff
@@ -0,0 +1,2 @@
+def rule(_):
+    return True
```

Comment:
Scheduled queries interact weirdly with scheduled rules sometimes. Basically a scheduled query can do 100% of the logic of whether or not an alert should fire, but it can't actually fire alerts. So instead you apply a scheduled rule to it that just returns true on anything the scheduled query returns. This is a generic return true function that can be used by scheduled queries that follow that pattern.